test/docs/OpenJ9TestUserGuide.md

```diff
@@ -109,6 +109,13 @@ Please read [DependentLibs.md](./DependentLibs.md) for details.
       - level:   [sanity|extended] (extended default value)
       - group:   [functional|system|openjdk|external|perf|jck] (required 
                  to provide one group per test)
+      - type:    [regular|native] (if a test is tagged with native, it means this
+                 test needs to run with test image (native test libs); 
+                 NATIVE_TEST_LIBS needs to be set for local testing; if Grinder is used, 
+                 native test libs download link needs to be provided in addition to SDK 
+                 download link in CUSTOMIZED_SDK_URL. For details, please refer to 
+                 https://github.com/AdoptOpenJDK/openjdk-tests/wiki/How-to-Run-a-Grinder-Build-on-Jenkins; 
```

Comment:
Could we use the markdown link? For example:

please refer to [How-to-Run-a-Grinder-Build-on-Jenkins](https://github.com/AdoptOpenJDK/openjdk-tests/wiki/How-to-Run-a-Grinder-Build-on-Jenkins)